lib/dialogs/trash-tag-confirmation/index.tsx

```diff
@@ -0,0 +1,60 @@
+import actions from 'lib/state/actions';
```

Comment:
it probably feels arbitrary but this, for consistency sake, belongs below third-party imports. it looks like your IDE might have auto-included this and the imports in line 5. we can replace both import statements with the more standard-in-this-app `import actions from '../../state/actions'` and then call using the fully-qualified action access, like `actions.ui.closeDialog`

Reply:
Thanks for catching this!